README.md

```diff
@@ -48,8 +52,7 @@ Or, if you'd like to contribute, send us a [pull request] and refer to our [code
 You can also reach us on our [Discord channel].
 
 For more information, check out the detailed guide on the
-[Google Developers site][devsite-guide]. You can also view the generated
-[reference docs][javadoc] for a full list of classes and their methods.
```

Comment:
I don't think `[javadoc]` was intended to be removed here?

Reply:
That was intentional since the text is in more prominent in its own section now. We can add that back if you think it'll help?

Reply:
Oh, sorry - somehow I missed that looking at the diff, LGTM.